spec/controllers/dashboard/firms_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+module Dashboard
+  RSpec.describe Dashboard::FirmsController, type: :controller do
+    let(:principal) { FactoryGirl.create(:principal) }
```

Comment:
I already discussed this with @munckymagik and was happy for `FactoryGirl` to be left in there so it's easy to pick out by eye or by search where the fixtures are being created